config/singularity-eos-config.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#cmakedefine PORTABILITY_STRATAGY_KOKKOS
+
+#cmakedefine SINGULARITY_INVERT_AT_SETUP
+#cmakedefine SINGULARITY_EOS_SKIP_EXTRAP
+
+#cmakedefine SINGULARITY_USE_FMATH_ORDER @SINGULARITY_USE_FMATH_ORDER@
+#cmakedefine SINGULARITY_USE_SINGLE_LOGS
+#cmakedefine SINGULARITY_USE_TRUE_LOG_GRIDDING
```

Comment:
:+1: nice addition. This will be useful to build on without all the target compiles flying around (in the future of course)